features/0420-rich-schemas-common/README.md

```diff
@@ -0,0 +1,251 @@
+# 0420: Rich Schema Objects Common
+- Author: [Alexander Shcherbakov](alexander.shcherbakov@evernym.com), [Brent Zundel](brent.zundel@evernym.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-13
+- Status Note: Part of proposed Rich Schema capabilities for credentials 
+- Start Date: 2020-02-05
+- Tags: [feature](/tags.md#feature), [rich-schemas](/tags.md#rich-schemas)
+
+
+## Summary
+
+A low-level description of the components of an anonymous credential ecosystem that supports rich schemas,
+W3C Verifiable Credentials and Presentations, and correspondingly rich presentation requests. 
+
+Please see [0119: Rich Schema Objects](https://github.com/hyperledger/indy-hipe/tree/master/text/0119-rich-schemas) 
```

Comment:
Were these links to HIPE 0119 intended to be to the Aries RFC version?
https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0250-rich-schemas

Reply:
Thank you! Fixed by pointing to Arfies RFCs instead.